grails-app/views/report/showTransactionReport.gsp

```diff
@@ -51,12 +51,10 @@
 									<warehouse:message code="report.startDate.label"/>
 								</label>
 
-								<g:jqueryDatePicker id="startDate"
-													name="startDate"
+								<g:jqueryDatePicker name="startDate"
 													cssClass="filter"
-													value="${command?.startDate }"
-													format="MM/dd/yyyy"
-													autocomplete="off"/>
+													value="${command?.startDate}"
+                                />
```

Comment:
the changes to `g:jqueryDatePicker` that I made are backwards compatable and so the change to this gsp is actually not required. However I did it to demonstrate the minimal set of fields that we'd need to provide. Everything else defaults correctly.